src/Nevermind/Nevermind.Core/TransactionReceipt.cs

```diff
@@ -17,5 +17,6 @@ public class TransactionReceipt
         public long GasUsed { get; set; }
         public Bloom Bloom { get; set; }
         public LogEntry[] Logs { get; set; }
+        public Address Recipient { get; set; }
```

Comment:
need to learn more - this I guess is only in the network version but not in the one created by EVM